adapta/security/clients/_base.py

```diff
@@ -71,3 +71,9 @@ def get_pyarrow_filesystem(self, path: DataPath, connection_options: Optional[Di
         :param connection_options: Optional connection options to use instead of auto-discovery.
         :return:
         """
+
+    @abstractmethod
+    def initialize_session(self, session_callable=None) -> "AuthenticationClient":
```

Comment:
Missing type hints

Reply:
Missing indeed!
Added in [3943e2d](https://github.com/SneaksAndData/adapta/pull/437/commits/3943e2dda0291afba169f072092af8a0ff6741b7)
Thank you for pointing it out 🙂